src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -33,14 +34,16 @@ final class LocoProvider implements ProviderInterface
     private HttpClientInterface $client;
     private LoaderInterface $loader;
     private LoggerInterface $logger;
+    private TranslatorBagInterface $translatorBag;
     private string $defaultLocale;
     private string $endpoint;
 
-    public function __construct(HttpClientInterface $client, LoaderInterface $loader, LoggerInterface $logger, string $defaultLocale, string $endpoint)
+    public function __construct(HttpClientInterface $client, LoaderInterface $loader, LoggerInterface $logger, TranslatorBagInterface $translatorBag, string $defaultLocale, string $endpoint)
```

Comment:
changes on constructors must be BC compliant, that's not the case in the PR right now

Reply:
Fixed, thanks!